cognite/powerops/main.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import Annotated
+
+import typer
+
+app = typer.Typer()
+
+
+@app.command(
+    "plan",
+    help="Compare CDF resource creation between desired state (configuration files)"
+    " and remote state (resources in CDF) and print differences",
+)
+def plan(
+    path: Annotated[Path, typer.Argument(help="Path to configuration files")],
+):
+    typer.echo(f"Running plan on configuration files located at {path}")
+
+
+@app.command("apply", help="Apply differences between desired and actual state. Writes desired state to CDF")
```

Comment:
Add parenthesis here as well. Just to be very explicit what is desired and actual state. 
```suggestion
@app.command("apply", help="Apply the changes from the configuration files to the data model in CDF")
```